### ISSUE TITLE
[CoE Starter Kit - BUG] Top 20 Environments on second Tab is Blank

### ISSUE THREAD

**Issue by NPrice99** [microsoft/coe-starter-kit#3634]
### Describe the issue

The "Top 20 Environments by App Count" in the "Overview - Power Apps" tab is blank even though the Total Environments Count is 5

### Expected Behavior

Up until recently this was populated with Environments

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.44

### What app or flow are you having the issue with?

Production _COEDashBoard_September2022

### Steps To Reproduce

1. Display the Production_COEDashboard_September2022

### Anything else?

![image](https://user-images.githubusercontent.com/8484048/189312669-08785c9b-d1ff-43f9-8a10-ffa50e1a7334.png)

**Comment by JMG74** [comment]
Hi Team, in fact the 5 tabs called "Overview ..." are blank.
I found that the problem relates to the page filter concerning field **admin_pvadeleted** (and when table **Virtual Agent** is empty). This filter is currently set to "false" but when I change it to "Select all", I'm getting the correct results !! This seems to be a good workaround in my context, but please fix this issues asap! 
Thanks in advance.

**Comment by NPrice99** [comment]
Well Spotted ! - I can confirm that this workaround populates the Overview - PowerApps Top 20 Environments Table

**Comment by manuelap-msft** [comment]
Thanks for the investigation here @JMG74 - that made it easier for us to fix, the new file has been uploaded just now and should work
https://github.com/microsoft/coe-starter-kit/raw/main/CenterofExcellenceResources/Release/Collateral/CoEStarterKit/Production_CoEDashboard_September2022.pbit

I have updated each Overview - page to only filter out resources on this page, so on the Overview - Power Automate page it filters out flows etc.
![image](https://user-images.githubusercontent.com/55705669/189346423-453ea392-e3da-4fda-b085-a132b5d96c51.png)

Hope that works now - if you could test the new file, I'd appreciate that!

**Comment by JMG74** [comment]
Thanks Manuela for your very fast answer.
I did some tests and I have a good and a bad news...

Starting with the good news : I confirm that now all the "Top 20 Environments by..." are correct in the 5 tabs called "Overview...". Thanks !

Now the bad news : as a consequence of removing some of the filter expressions on the 5 tabs, the counters located at the top are now **wrong**. Look at the following picture where I collected the counters from the 5 tabs :
![image](https://user-images.githubusercontent.com/13162921/189375852-58436e39-203a-4a59-ad39-8cd7ec0288fa.png)

By the way, do we really need getting the same counters on all "Overview..." tabs ?? Personally, I would prefer getting the counters of apps only on "Overview - Power Apps" tab, counters of flows only on "Overview - Power Automate" tab, and counters of chatbots only on "Overview - Chatbots" tab.  

Looking forward for your feedback. Thanks in advance

**Comment by manuelap-msft** [comment]
Thanks for letting us know and the detailed investigation - very helpful!

I agree, when we introduced this counter section we had a lot less resources in the Power BI and now it's getting a bit crowded and it's a lot of duplication! We'll work on updating the Overview pages to only show the counters of the overview page that you're on - and perhaps even dedicating a bit more space to that, e.g. on Power Apps show more detailed counts for model driven apps, canvas apps, custom pages etc. 

I've added this to our backlog for October!
